elements/pfe-sass/mixins/_components.scss

```diff
@@ -193,11 +193,12 @@
 /// Accordion chevrons
 /// ===========================================================================
 
-@mixin pfe-chevron($state: open, $position: after, $size: .4em, $position: $size) {
+@mixin pfe-chevron($state: open, $position: after, $size: .5em, $offset: $size) {
```

Comment:
i think the default here needs to go back to .4em since that controls the size. I think you have the positioning covered with the $offset argument

Reply:
Fixed